### PR TITLE
Legacy TPBs and playbooks to DeprecatedContent

### PR DESCRIPTION
**Pull request by dantavori**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
Moved TPBs and playbooks from Legacy to DeprecatedContent.

**The following TPBs were not moved because they are test playbooks for non-deprecated integrations:**
- Nmap - Test
- MxToolbox-test
- ThreatExchange-test
- PerceptionPoint Test
- OpenPhish Test Playbook
- TestIsValueInArray
- Google Safe Browsing Test
- DocumentationTest

**The following playbooks were not moved because they are dependencies for other playbooks/scripts which are still on Legacy (or other packs):**
- TestWordFileToIOC (playbooks in Carbon Black Live Response)
- Carbon Black Rapid IOC Hunting (playbook in Legacy)
- ExposeIncidentOwner-Test (script in Legacy)
- TestFileCreateAndUpload (script in Legacy)
- Send Investigation Summary Reports - Test (playbook in Legacy)
- TestExtractHTMLTables (script in Legacy)
- HTTPListRedirects Basic Test (script in Legacy)
- Send Investigation Summary Reports (playbook in Legacy)
- LanguageDetect-Test (script in Legacy)
- ADGetComputer (script in Legacy)
- listExecutedCommands-Test (script in Legacy)
- Block IP - Generic (script in Legacy)
- Endpoint Enrichment - Generic (script in Legacy)

### CODE REVIEW

**Comment by dantavori** [comment]
after we merge this one I'll pull from master the other three branches [(#1](https://github.com/demisto/content/pull/6971), [#2](https://github.com/demisto/content/pull/6972), [#3](https://github.com/demisto/content/pull/6973))

**Comment by liorblob** [comment]
Some of the test playbooks there were not in `conf.json` nor in their integration yml (`no test`). We can maybe move them to their integrations and skip them in `conf.json` or just put them in deprecated content.

File: Packs/DeprecatedContent/pack_metadata.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "description": "Deprecated Cortex XSOAR content pack.",
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Utilities"
+    ],
+    "tags": [],
```
Comment (IkaDemisto) [inline Packs/DeprecatedContent/pack_metadata.json:12]: Should be labeled somehow as `deprecated`, maybe add to tags

**Comment by IkaDemisto** [comment]
@dantavori can we move the `deprecated` integration from `Legacy` pack also?

**Comment by ShahafBenYakir** [comment]
@dantavori Any reason why we decided to deprecate the extrahop - ticket tracking PB?
https://github.com/demisto/content/pull/6974/files#diff-18e840d30236069ced71aebc9ce1d70d
The vendor is asking.

**Pull request merged**